inc/sitemaps/interface-sitemap-cache-data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @package WPSEO\XML_Sitemaps
+ */
+
+/**
+ * Cache Data interface
+ */
+interface WPSEO_Sitemap_Cache_Data_Interface {
+
+	/** Status for normal, usable sitemap. */
+	const OK = 'ok';
+
+	/** Status for unusable sitemap. */
+	const ERROR = 'error';
+
+	/** Status for unusable sitemap because it cannot be identified. */
```

Comment:
Because we cannot force the values passed to be either one of the two options, I wanted to catch all invalid input to match an invalid status but be able to identify it was entered incorrectly.